app/stacks/uk-west/back-office/app-service.tf

```diff
@@ -49,12 +49,16 @@ module "app_services" {
   servicebus_topic_register_nsip_subscription_id                                      = azurerm_servicebus_topic.register_nsip_subscription.id
   servicebus_topic_deadline_submission_topic_id                                       = azurerm_servicebus_topic.deadline_submission_topic.id
   deadline_submissions_topic_name                                                     = azurerm_servicebus_topic.deadline_submission_topic.name
-  tags                                                                                = local.tags
-  employee_topic_id                                                                   = azurerm_servicebus_topic.employee.id
-  feature_odw_subscription_enabled                                                    = true
-  back_office_documents_system_topic_name                                             = azurerm_eventgrid_system_topic.back_office_documents_system_topic.name
-  notify_subscribers_function_gov_notify_template_id                                  = var.notify_subscribers_function_gov_notify_template_id
-  applications_front_office_web_url                                                   = var.applications_front_office_web_url
+  #service_bus_appeals_bo_case_topic_id                                                = azurerm_servicebus_topic.appeal_bo_case.id
+  #service_bus_appeals_bo_document_topic_id                                            = azurerm_servicebus_topic.appeal_bo_document.id
```

Comment:
Do these need to be populated or can we just delete them?

Reply:
I commented them out to avoid warnings, but I think eventually Appeals FO will use them. Probably leave for the moment I'll follow up.